server/www/flightinsure.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+    <!-- Bootstrap CSS -->
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+
+    <meta charset="utf-8">
+    <title>Flight Insurance</title>
+  </head>
+  <body>
+    <h3>Would you like to add insurance to your flight?</h3>
+    <script type="text/javascript" src="js/bootstrap.bundle.js"></script>
+    <script type="text/javascript" src="js/bootstrap.min.js"></script>
```

Comment:
these bootstrap files aren't in the repo. they might be in ur local computer, but haven't been committed so they aren't included in this pull request && thus don't actually load when i test. did u mean to leave these script tags here? if so u should also commit those bootstrap files, if not u should remove these

Reply:
I think I meant to remove them but I will check

Reply:
Oh that page was suppose to get deleted, sorry about that

Reply:
ok cool, merging now